src/components/FullScreenPage.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactNode } from 'react';
+import styled from 'styled-components';
+
+
+interface FullScreenLayoutProps {
+    children: ReactNode
+  }
+
+
+const FullScreenPage: React.FC<FullScreenLayoutProps> =({children})=>{
+    return(
+        <Section>
+            <Title> {children}</Title>
```

Comment:
Delete the title component from here. 
This component is only a wrapper for other components.